ControlRoomApplication/ControlRoomApplication/Controllers/Communications/pushNotification.cs

```diff
@@ -52,15 +59,15 @@ public static bool sendToAllAdmins(String titleText, String bodyText, bool testf
             };
 
             // Send a message to the device corresponding to the provided
-            string response = FirebaseMessaging.DefaultInstance.SendAsync(message).Result;
-
+           // string response = FirebaseMessaging.DefaultInstance.SendAsync(message).Result;
```

Comment:
Should we leave this commented out on dev/prod or is this something we forgot to remove in for testing?